### PR TITLE
`clj-rq` v0.2.x release

### LINKED ISSUE #11: write blogpost about clj-rq's release `v0.2.0`

### PR DESCRIPTION
**Pull request by J0sueTM**
closes https://github.com/moclojer/moclojer.com/issues/11

## Summary by CodeRabbit

- **New Features**
	- Introduced a blog post detailing the dynamic self-generation of the `clj-rq` library for easier interaction with Redis.
	- Provided installation instructions and usage examples for the library and its new `components` bundle.
- **Documentation**
	- Added comprehensive information about the library's motivation, functionality, and maintenance improvements.

### CODE REVIEW

File: content/blog/clj-rq-v0.2-release.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+---
+title: "Our lib `clj-rq` now generates itself dynamically"
+date: 2024-08-23T01:00:00+10:00
+tags: ["foss", "clj-rq", "release"]
+images: TODO
```
Comment (J0sueTM) [inline content/blog/clj-rq-v0.2-release.md]: Add images. Reminding myself.

File: content/blog/clj-rq-v0.2-release.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+```clojure
+[com.moclojer/components "0.1.1"]
+```
+
+> Be sure to check the latest versions incase you're reading this in the future.
+
+---
+
+Thanks for reading. You can find more about moclojer, [who we are, what we do, what we eat](https://www.youtube.com/watch?v=XPxOI8YyKtw) and more at our [home page](https://moclojer.com), our [github](https://github.com/moclojer), and maybe our [product](https://app.moclojer.com?utm_source=blog&utm_medium=post&utm_campaign=clj-rq-v0.2.0).
```
Comment (J0sueTM) [inline content/blog/clj-rq-v0.2-release.md:126]: @avelino Do the utm queries look alright?

File: content/blog/clj-rq-v0.2-release.md (hunk 1 above)
Comment (avelino) [reply content/blog/clj-rq-v0.2-release.md]: what do you think?

e5e61e734065bf1b98be7e08213caf0ce8be31b8

Comment (J0sueTM) [reply content/blog/clj-rq-v0.2-release.md]: looks great. I edited it up to remove the janky text
clj-rq-now-generates-itself-dynamically

Comment (avelino) [reply content/blog/clj-rq-v0.2-release.md]: very good!

**Pull request merged**